Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -294,6 +274,83 @@ private void Awake()
         /// </summary>
         private void OnEnable()
         {
+            EnableAllManagers();
+        }
+
+        /// <summary>
+        /// The MonoBehaviour Update event, which is then circulated to all active managers
+        /// </summary>
+        private void Update()
+        {
+            UpdateAllManagers();
```

Comment:
I approve of this abstraction,  I've been doing it more and more in refactoring.